.vsts.pipelines/jobs/ci-linux_bootstrap.yml

```diff
@@ -0,0 +1,96 @@
+parameters:
+  job: null
+  matrix:
+    Production: {}
+  pool:
+    name: Hosted Ubuntu 1604
+  imageName: null
+  dependsOn: ''
+
+jobs:
+- job: ${{ parameters.job }}
+  dependsOn: ${{ parameters.dependsOn }}
+  strategy:
+    matrix: ${{ parameters.matrix }}
+  pool: ${{ parameters.pool }}
+  timeoutInMinutes: 270
+  variables:
+    # Prefix to distinguish artifacts from different legs.
+    artifactName: ${{ format('{0} $(type)', parameters.job) }}
+    # Use ":z" to set selinux flag for sharing in build-owned root dir. https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label
+    docker.agentSrc.map: -v $(Build.SourcesDirectory):/agentSrc:z
+    docker.agentSrc.work: -w /agentSrc
+    docker.drop.map: -v $(dropDirectory):/drop:z
+    docker.logs.map: -v $(dropDirectory)/logs:/logs:z
+    docker.root.map: -v $(rootDirectory):/root:z
+    docker.run: docker run --rm
+    docker.bst.map: -v $(System.ArtifactsDirectory)/boostrap_root:/bst:z
+    docker.bst.work: -w /bst
+    dropDirectory: $(stagingDirectory)/drop
+    imageName: ${{ parameters.imageName }}
+    rootDirectory: $(Build.SourcesDirectory)/..
+    stagingDirectory: $(rootDirectory)/bst/staging
+    tarballName: tarball_$(Build.BuildId)
+    SOURCE_BUILD_SKIP_SUBMODULE_CHECK: true
+    # Default type, can be overridden by matrix legs.
+    type: Production
+
+  steps:
+  - template: ../steps/docker-cleanup-linux.yml
+
+  # Create working directory and copy source into it.
```

Comment:
This isn't doing what the comment says.  I'm not sure if this step is needed.  Do you use /root/sb/tarball later in the job?  I don't see it being used anywhere.

Reply:
Right. I am using the one from `DownloadArtifacts` Task. Thanks for the catch. 👍